packages/unleash-frontend-next/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "unleash-frontend-next",
+  "description": "unleash your features",
+  "version": "1.0.0",
+  "keywords": [
+    "unleash",
+    "feature toggle",
+    "feature",
+    "toggle"
+  ],
+  "files": [
+    "public"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "ssh://git@github.com:finn-no/unleash.git"
+  },
+  "bugs": {
+    "url": "https://github.com/finn-no/unleash/issues"
+  },
+  "engines": {
+    "node": "6"
+  },
+  "scripts": {
+    "build": "webpack -p",
+    "start": "webpack-dev-server --config webpack.config.js --hot --progress --colors --port 3000",
+    "test": "echo 'no test'",
+    "test:ci": "npm run test",
+    "prepublish": "npm run build"
+  },
+  "main": "./index.js",
+  "dependencies": {
+    "immutability-helper": "^2.0.0",
+    "react": "^15.3.1",
+    "react-addons-css-transition-group": "^15.3.1",
+    "react-dom": "^15.3.1",
+    "react-redux": "^4.4.5",
+    "react-router": "^2.8.0",
```

Comment:
because this is what you get when you do `npm install react-router --save`. might be worth switching..?